src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/packagedeclaration/InputPackageDeclarationDiffDirectoryAtParent.java

```diff
@@ -1,8 +1,8 @@
 package com.puppycrawl.tools.checkstyle.checks.coding;
 
-class InputPackageDeclaration {
+public class InputPackageDeclarationDiffDirectoryAtParent {
     public String value;
-    
+
```

Comment:
@romani 
Should I remove unnecessary spacing in such way as it was done previously?